core/src/main/resources/spline.default.properties

```diff
@@ -90,13 +90,16 @@ spline.postProcessingFilter.composite.className=za.co.absa.spline.harvester.post
 # The classes must have a constructor with a single parameter of type {org.apache.commons.configuration.Configuration}
 #
 # Example:
-#   to enable password replacing in the data source URIs add "dsUriPasswordReplace" filter to the list:
-#spline.postProcessingFilter.composite.filters=dsUriPasswordReplace
+#   to enable password replacing in the data source URIs add "dsPasswordReplace" filter to the list:
+#spline.postProcessingFilter.composite.filters=dsPasswordReplace
 spline.postProcessingFilter.composite.filters=
 
 # This filter replaces password occurrences in the data source URIs
-spline.postProcessingFilter.dsUriPasswordReplace.className=za.co.absa.spline.harvester.postprocessing.DataSourcePasswordReplacingFilter
-spline.postProcessingFilter.dsUriPasswordReplace.replacement=*****
-spline.postProcessingFilter.dsUriPasswordReplace.regexes=\
-  [?&;]password=([^&;]*)(?\=[&;])? ,\
+spline.postProcessingFilter.dsPasswordReplace.className=za.co.absa.spline.harvester.postprocessing.DataSourcePasswordReplacingFilter
+spline.postProcessingFilter.dsPasswordReplace.replacement=*****
+spline.postProcessingFilter.dsPasswordReplace.nameRegexes=\
+  (?i)password ,\
+  (?i)passphrase
+spline.postProcessingFilter.dsPasswordReplace.valueRegexes=\
+  (?i)[?&;](?:password|passphrase)=([^&;]*)(?\=[&;])? ,\
```

Comment:
What exactly is `?\=` dooing in `(?\=[&;])?` ? 

Also, this regexp is not covered by the test.